.github/workflows/release.yaml

```diff
@@ -90,25 +90,35 @@ jobs:
 
       - uses: ko-build/setup-ko@v0.6
         with:
-          version: v0.14.1
+          version: v0.15.1
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.2.0
+        with:
+          cosign-release: "v2.2.2"
 
       - name: Prepare
         run: |
           echo "BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_ENV
+          echo "REPOS"="quay.io/grafana-operator/grafana-operator" "ghcr.io/${{ github.repository }}" >> $GITHUB_ENV
```

Comment:
Is this a valid list declaration? Doesn't it need to be wrapped in parentheses? 
I think i'm seeing this issue in my local testing, where the iterator wants to reference `ghcr` but keeps referencing `quay` causing a "cannot open file/doesn't exist" in `./image-digest-${i}` reference

Reply:
So I played around with this in a separate PR to make sure I did it correctly https://github.com/NissesSenap/grafana-operator/pull/26/files

https://github.com/NissesSenap/grafana-operator/actions/runs/7238546246/job/19719472486
![image](https://github.com/grafana/grafana-operator/assets/14774899/6517758e-3d81-4f5b-be33-dee97e023b7a)

So I think it should be okay. Do you have an example where you have triggered it with github actions?
I haven't tried this version since I don't have a `quay.io` registry anymore. It got screwed up when RedHat started to enforce redhat accounts.

Reply:
Not sure if you can access this: https://github.com/HubertStefanski/grafana-operator/actions/runs/7302136053/job/19900190699

Reply:
![image](https://github.com/grafana/grafana-operator/assets/35736504/ad93f5da-2ce6-4705-8e4b-5a356382a27f)